### DISCUSSION TITLE
"Shutting down the letsblock.it project and its official instance"

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uBlock-issues#3194] [General]
https://letsblock.it/  
https://github.com/letsblockit/letsblockit/commit/c6d8ca74d5226a7507eeeaaaf787e3c710198a1e#diff-a649e8e37a1f55c5b5b9a6cdf4fa5636c06f58b811edd92ac66e44dcf27cfb33

Support should be removed from the subscriber script.

@gorhill

**Comment by gorhill** [comment]
https://github.com/gorhill/uBlock/commit/98a600698e25533ea1bf9c228885791cee5fa636
